commcare/src/main/resources/velocity.templates/task-data-provider.vm

```diff
@@ -163,7 +163,7 @@
                                     "displayName" : "commcare.case.field.caseName",
                                     "fieldKey" : "caseName"
                                 },
-                                #foreach($caseField in $entry.value)
+                                #foreach($caseField in $entry.getApplicationName())
```

Comment:
@sebbrudzinski There is a field 'caseProperties' in CommcareModuleJSON. From what I understood, this is qualified as the collection of case fields. There's nothing explicitely named case fields in CommCare.